sci-calculators/bc-gh/bc-gh-2.5.3.ebuild

```diff
@@ -0,0 +1,19 @@
+# Copyright 2019 Gentoo Authors
```

Comment:
You can use repoman to commit so it fixes this copyright header for you automatically.

Reply:
Okey I will do it :) Thank You for advice :3